src/components/MessageCard/utils/MessageCard.hooks.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+const TRANSITION_DURATION = 200
+
+/**
+ * Handles the logic for resizing survey buttons
```

Comment:
I think this hook is here because you are waiting for the animation to finish. Can you add a little note about that? (or if I'm mistaken, a note about the real why 🙂)